src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -1,19 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@iSazonov, your last commit had 3 failures in `PowerShell-CI-windows`
Error occurred in test script 'D:\a\1\s\test\powershell\engine\Basic\Assembly.LoadNative.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\engine\Basic\Assembly.LoadNative.Tests.ps1:64 char:16
+         $PSHOME\$sourceDllName | Should -BeNullOrEmpty
+                ~~~~~~~~~~~~~~~
Unexpected token '\$sourceDllName' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\engine\Basic\Assembly.LoadNative.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\engine\Basic\Assembly.LoadNative.Tests.ps1:64 char:16
+         $PSHOME\$sourceDllName | Should -BeNullOrEmpty
+                ~~~~~~~~~~~~~~~
Unexpected token '\$sourceDllName' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\engine\Basic\Assembly.LoadNative.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\engine\Basic\Assembly.LoadNative.Tests.ps1:64 char:16
+         $PSHOME\$sourceDllName | Should -BeNullOrEmpty
+                ~~~~~~~~~~~~~~~
Unexpected token '\$sourceDllName' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```